server/plugin/command.go

```diff
@@ -121,18 +121,18 @@ func (p *MatterpollPlugin) executeCommand(args *model.CommandArgs) (string, *mod
 	}
 
 	var newPoll *poll.Poll
-	var err error
+	var errMsg *poll.ErrorMessage
 	if len(o) == 0 {
-		newPoll, err = poll.NewPoll(creatorID, q, []string{defaultYes, defaultNo}, s)
+		newPoll, errMsg = poll.NewPoll(creatorID, q, []string{defaultYes, defaultNo}, s)
 	} else {
-		newPoll, err = poll.NewPoll(creatorID, q, o, s)
+		newPoll, errMsg = poll.NewPoll(creatorID, q, o, s)
 	}
-	if err != nil {
+	if errMsg != nil {
 		appErr := &model.AppError{
 			Id: p.LocalizeWithConfig(userLocalizer, &i18n.LocalizeConfig{
 				DefaultMessage: commandErrorInvalidInput,
 				TemplateData: map[string]interface{}{
-					"Error": err.Error(),
+					"Error": p.LocalizeErrorMessage(userLocalizer, errMsg),
```

Comment:
Just noticed that I'm wrapping an error here :man_shrugging:

Reply:
I don't have a good idea for this...
Do you intend to do further changes for this? If not, I'll approve this PR.

Reply:
No, let's just go with this for now.